### PR TITLE
Fixes #265. Adds usage docs and require statements to some functions

### LINKED ISSUE #265: quite a few of the functions are missing usages docs or require statements
For example:

`string.esc` is missing usage documentation
https://canjs.com/doc/can-util/js/string/string.esc.html

`stringToAny` is missing a `require("can-util/js/string-to-any")`
https://canjs.com/doc/can-util/js/string-to-any/string-to-any.html

### PR DESCRIPTION
**Pull request by mjstahl**

### CODE REVIEW

File: dom/frag/frag.js [hunk 1]
```diff
@@ -24,6 +24,8 @@ Convert a String, HTMLElement, documentFragment, or contentArray into a document
 
 ContentArrays can be used to combine multiple HTMLElements into a single document fragment.  For example:
 
+    var frag = require("can-util/dom/frag/");
```
Comment (chasenlehara) [inline dom/frag/frag.js]: We avoid the short (StealJS-friendly) path for users of other module loaders which don’t support this syntax.

File: js/string/string.js [hunk 2]
```diff
@@ -53,6 +53,12 @@ var string = {
 	 * @signature `string.esc(content)`
 	 * @param  {String} content a string
 	 * @return {String}         the string safely HTML-escaped
+	 * 
+	 * ```
```
Comment (chasenlehara) [inline js/string/string.js]: I would explicitly mark any code blocks with ```js

**Review by chasenlehara — CHANGES_REQUESTED** [review]
Looks great! I left a couple comments on some things that could be improved throughout the files, but this looks good to merge after those are fixed.

File: js/string/string.js (hunk 2 above)
Comment (mjstahl) [reply js/string/string.js]: fixed.

File: dom/frag/frag.js (hunk 1 above)
Comment (mjstahl) [reply dom/frag/frag.js]: Updated this, and the other `require` lines

**Review by chasenlehara — APPROVED** [review]

**Pull request merged**